Language_Parser/itemDictionary.json

```diff
@@ -35,13 +34,10 @@
     "mattress": ["mattress", "bed"],
     "mouse": ["mouse"],
     "noise": ["noise", "tapping"],
-    "painting": ["painting", "portrait", "paintings", "portraits", "frame", "frames"],
+    "painting": ["painting", "portrait", "paintings", "portraits", "frame", "frames", "firstpainting", "secondpainting", "thirdpainting"],
     "pillow": ["pillow", "pillows"],
     "pills": ["pills", "pill", "pillbottle", "bottle", "medicine", "medicinebottle", "medication"],
-    "polaroid": ["polaroid", "polaroidphoto", "picture"],
-    "polaroid1": ["polaroid1"],
-    "polaroid2": ["polaroid2"],
-    "polaroid3": ["polaroid3"],
+    "polaroid": ["polaroid"],
```

Comment:
Just curious, do we no longer want "polaroidphoto" and "picture" to reference "polaroid"?

Reply:
Do we want "oldpolaroid" or "dustypolaroid" to be captured here?

Reply:
We can include "picture" probably. The functions will break if you include "oldpolaroid" and "dustypolaroid" though, since the helper functions translate "polaroid" -> [specific polaroid], including those two specific-polaroid terms essentially changes the flow to [specific polaroid] -> "polaroid". Found that out the hard way.

Reply:
Polaroids will probably also be tweaked some more when I get to the third one, if it matters.